pkpdapp/pkpdapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,80 @@
+#
+# This file is part of PKPDApp (https://github.com/pkpdapp-team/pkpdapp) which
+# is released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+# Generated by Django 3.0.7 on 2020-12-04 16:50
+# flake8: noqa
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BiomarkerType',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(help_text='name of the biomarker type', max_length=100)),
+                ('unit', models.CharField(choices=[('mg', 'type1')], help_text='units for the value stored in :model:`pkpdapp.Biomarker`', max_length=2)),
```

Comment:
Does `type1` mean something special? If not, wouldn't it exceed the given max char length? (Again, I'm sure my question is stupid!)

Reply:
perfectly correct, mistake on my part, I've fixed this now

Reply:
fyi, type1 was just a placeholder entry cause I'm not sure of the possible units that will be required here